projects/golang/build.sh

```diff
@@ -0,0 +1,54 @@
+# Get golang
+wget https://dl.google.com/go/go1.11.1.linux-amd64.tar.gz
+tar zxvf go1.11.1.linux-amd64.tar.gz
+export GOROOT=`realpath go`
+export GOPATH=$GOROOT/packages
+mkdir $GOPATH
+export PATH=$GOROOT/bin:$PATH
+
+mkdir -p $GOPATH/src/github.com/dvyukov/
+cd $GOPATH/src/github.com/dvyukov/
+git clone https://github.com/dvyukov/go-fuzz-corpus
+cd -
+go get github.com/guidovranken/libfuzzer-go/...
+
+go build github.com/guidovranken/libfuzzer-go/go-fuzz-build
+
+function compile_fuzzer {
+    fuzzer=$(basename $1)
+    echo "Fuzzer is $fuzzer"
+    $CC $CFLAGS -Wall -Wextra $GOPATH/src/github.com/guidovranken/libfuzzer-go/C/main_libFuzzer_extra_counters.c -g -O3 -c -o main.o
+    ./go-fuzz-build -o $fuzzer.a github.com/dvyukov/go-fuzz-corpus/$fuzzer
```

Comment:
maybe add some comments on how the code gets instrumented here. 
I assume this is source-to-source thing that go-fuzz uses, but not sure, and other readers won't know.